editor/js/reaction.js

```diff
@@ -65,7 +65,7 @@ async function init(fileName, index) {
   loadReaction(reaction);
   clean();
   // Trigger reaction-level validation.
-  validateReaction();
+  setTimeout(validateReaction, 1000);
```

Comment:
This only happens once, on startup, right?

Reply:
Yes